config/dispatcher/dispatcher.ex

```diff
@@ -301,12 +301,12 @@ defmodule Dispatcher do
     Proxy.forward conn, path, "http://cache/configs/"
   end
 
-    match "/numac-numbers/*path", @any do
-      Proxy.forward conn, path, "http://cache/numac-numbers/"
-    end
+  match "/numac-numbers/*path", @any do
+    Proxy.forward conn, path, "http://cache/numac-numbers/"
+  end
 
-  match "/publication-types/*path", @any do
-    Proxy.forward conn, path, "http://cache/publication-types/"
+  match "/publication-modes/*path", @any do
```

Comment:
Was niet bij in de story gespecifieerd, maar kan je hier nog `get` van maken ipv `match`? Dan is is 't meteen gebeurd en weet je dat voor de volgende. Zie ook https://github.com/kanselarij-vlaanderen/kaleidos-project/pull/179#discussion_r610828678